### ISSUE TITLE
Accessibility - Search Results Links Missing Underline

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Ilab-do-good#83]
The links on the search results pages/screen are missing an underline for both iOS and Android. Having the underline if you are colorblind helps tell people that it is a link and not plain text.

Same applies to the website too.

**Comment by adriennerlong** [comment]
From ICF: Unfixed.  Current known solution interferes with other accessibility aspects.

**Comment by jfc3-dol** [comment]
The iOS search results links now have an underline under the text, so people know they are links and not plain text.

Will close issue once I can check that both the Android version and the web version have underlines too.

**Comment by jfc3-dol** [comment]
The search results links now have underlines for the Android application.

Removing the Android label and going to check the website version once that is ready for me to test.

**Comment by jfc3-dol** [comment]
The search results links now have underlines for the web application.

Removing the web label and going to close the issue
